scripts/scripts.js

```diff
@@ -9,13 +9,35 @@ import {
   decorateBlocks,
   decorateTemplateAndTheme,
   getMetadata,
+  toClassName,
   waitForLCP,
   loadBlocks,
   loadCSS,
 } from './lib-franklin.js';
 
 const LCP_BLOCKS = []; // add your LCP blocks to the list
 
+const AUDIENCES = { // add your custom audiences to the list
+  mobile: () => window.innerWidth < 600,
+  desktop: () => window.innerWidth >= 600,
+};
+
+/**
+ * Gets all the metadata elements that are in the given scope.
+ * @param {String} scope The scope/prefix for the metadata
+ * @returns an array of HTMLElement nodes that match the given scope
+ */
+export function getAllMetadata(scope) {
```

Comment:
I'm not very fond of circle imports....
What if we have a tiny script in the plugin itself which decides whether the plugin has to be loaded or not for the current page. I know it would have some small impact as it will request another .js file even when for pages without experimentation.
Another option would be to pass the audiences metadata in the `customOptions` of the plugin

Reply:
I think this is just a temporary issue. I'm hoping to contribute this into lib-franklin.js directly, and that would address your concern down the road essentially.

Moving anything back into `scripts.js` ultimately just makes it more complex to instrument the plugin, leaks logic over there and increases the risk devs don't do the setup properly. I'd rather have that circle import and make it easier for everyone else 😉